tox.ini

```
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27, py34, py35, py36, py37, py38, flake8, black, license
+envlist = py27 py34, py35, py36, py37, py38, flake8, black, license
```

Lost a comma here. Could also use tox's expansion syntax and drop Python 3.4 testing (which is no longer supported):
```suggestion
envlist = py{27,35,36,37,38}, flake8, black, license
```

---

Good point, was hesitating on doing that more concise setup, not entirely sure why, maybe readability. But in the end this feels indeed clearer, added!